lib/index.js

```diff
@@ -89,44 +88,81 @@ AdWords.prototype.page = function(page) {
  */
 
 AdWords.prototype.track = function(track) {
-  var id = this.options.conversionId;
+  var self = this;
   var props = track.properties();
-  var remarketing = this.options.remarketing;
-  var events = this.events(track.event());
+  var eventMappings = this.options.eventMappings;
   var revenue = track.revenue() || 0;
-  // Check if this is a whitelisted event for standalone remarketing tag
-  var whitelisted = this.options.whitelist.indexOf(track.event()) > -1;
-  var sentAlready = false;
 
-  each(function(label) {
-    delete props.revenue;
-    // Fire conversion tag
-    window.google_trackConversion({
-      google_conversion_id: id,
-      google_custom_params: props,
-      google_conversion_language: 'en',
-      google_conversion_format: '3',
-      google_conversion_color: 'ffffff',
-      google_conversion_label: label,
-      google_conversion_value: revenue,
-      google_remarketing_only: false // ensure this is a conversion tag
+  if (eventMappings) { // if eventMappings exists then the new metadata schema is being used
```

Comment:
This will always return true if the default is an array (even an empty one). Checking length !== 0 is likely better.

Reply:
I think this will work here because eventMappings doesn't exist in the current metadata schema and is being added via script, this check is to see whether the project is running on the old or new metadata so I think this will suffice?

Reply:
My concern would be then, earlier, where we define `.option('eventMappings', [])`.

Reply:
Can you clarify your concern there? To be honest not entirely sure what that line does I was just following convention there

Reply:
That line's job is effectively to say "use this.options.eventMappings || []". So it's there to make [] the default.

Reply:
ah ok I see now, checking length 0 won't work though because a lot of projects in the new metadata schema will have empty `eventMapping` lists